common/Makefile.common.mk

```diff
@@ -91,7 +91,7 @@ mirror-licenses: mod-download-go
 	@license-lint --mirror
 
 TMP := $(shell mktemp -d -u)
-UPDATE_BRANCH ?= "master"
+UPDATE_BRANCH ?= "release-1.17"
```

Comment:
This doesn't need to be in this one, but it's OK.

Reply:
Because this wasn't merged before Step 5 was done, I have the Step 5 stuff in the other PR to fix this up.